doc/help/configuring.asciidoc

```diff
@@ -349,12 +349,13 @@ bind_chained('<Escape>', 'clear-keychain', 'search')
 Avoiding flake8 errors
 ^^^^^^^^^^^^^^^^^^^^^^
 
-If you use an editor with flake8 integration which complains about `c` and `config` being undefined, you can use:
+If you use an editor with flake8 and pylint integration which complains about
+`c` and `config` being undefined or invalid, you can use:
 
 [source,python]
 ----
-c = c  # noqa: F821
-config = config  # noqa: F821
+c = c  # noqa: F821 pylint: disable=invalid-name,undefined-variable
+config = config  # noqa: F821 pylint: disable=invalid-name,undefined-variable
 ----
 
 For type annotation support (note that those imports aren't guaranteed to be
```

Comment:
For consistency, I think that the annotations on the block below should either be removed (implying you can add both) or added to both. 

Perhaps we can use the shorter form of pylint disable messages which use the error numbers instead of names to save space?

Otherwise, seems good to me!